package.json

```diff
@@ -61,22 +64,20 @@
     "extract-text-webpack-plugin": "^0.8.2",
     "file-loader": "^0.8.4",
     "filesaver.js": "^0.2.0",
-    "jest-cli": "^0.5.1",
+    "jest-cli": "^22.0.6",
     "json-stable-stringify": "^1.0.0",
     "leaflet-image": "^0.1.0",
     "normalize.css": "^3.0.3",
     "perfect-scrollbar": "^0.6.7",
     "postcss-cssnext": "^2.2.0",
     "postcss-import": "^7.1.0",
     "postcss-loader": "^0.7.0",
-    "react": "^0.14.2",
-    "react-addons-test-utils": "^0.14.3",
-    "react-bootstrap": "^0.29.5",
-    "react-dom": "^0.14.2",
+    "react": "^16.2.0",
+    "react-bootstrap": "^0.32.0",
+    "react-dom": "^16.2.0",
     "react-hot-loader": "^1.3.0",
     "react-loader": "^2.2.0",
-    "react-router": "^2.0.1",
-    "react-tabs": "^0.5.1",
+    "react-router": "^3.2.0",
```

Comment:
I'm getting an odd error: when I load the default portal, `/#/`, and then switch to a comparison portal by clicking on the routing links at the top, it sends me to `/compare/all_downscale_files`, which doesn't work, instead of `/#/compare/all_downscale_files`, which does. Suspect the 3.* version of react-router acts slightly differently than 2.*.

Reply:
[Seems to be a pretty simple fix](https://github.com/pacificclimate/climate-explorer-frontend/pull/114). Just adding the missing `/#` to the URLs does the trick. 

Though I did find [a new issue](https://github.com/pacificclimate/climate-explorer-frontend/issues/115) while testing that fix.

Reply:
Fixed.